Phoenix/hello/lib/hello_web/endpoint.ex

```diff
@@ -47,4 +47,17 @@ defmodule HelloWeb.Endpoint do
   plug Plug.Head
   plug Plug.Session, @session_options
   plug HelloWeb.Router
+
+  plug :introspect
```

Comment:
@samu975 Could you explain how a plug works?

Reply:
There exist two kinds of plugs, module plugs and function plugs.  A module plug is a module with two functions with some configurations. Both plugs work the same.  It receives a connections and options, transforms a connection and return the connection transformed.